README.md

```diff
@@ -414,6 +414,10 @@ To skip removing empty strings, set the `removeEmptyStrings` option to `false` w
 
 To skip adding automatic values, set the `getAutoValues` option to `false` when you call `insert` or `update`. This works only in server code.
 
+### Pick or omit from the attached schema
+
+To pick or omit fields from the schema for the operation, set the 'pick' or 'omit' option respectively to an array of schema field names. These options are mutually exclusive, so you cannot have both present in the options object at the same time.
+
```

Comment:
Would be nice to have an example here to prevent confusion as where to put the options. Also link to: https://github.com/aldeed/simple-schema-js#extracting-schemas